draft-ietf-quic-manageability.md

```diff
@@ -534,8 +534,9 @@ The QUIC wire image is not specifically designed to be distinguishable from
 other UDP traffic by a passive observer in the network.
 
 The only application binding defined by the IETF QUIC WG is HTTP/3
-{{?QUIC-HTTP}} at the time of this writing; however, many other applications
-are currently being defined and deployed over QUIC, so an assumption that all
+{{?QUIC-HTTP}} at the time of this writing; however, other applications
+are currently being defined and deployed over QUIC (e.g. DNS 
+{{?draft-ietf-dprive-dnsoquic}}), so an assumption that all
```

Comment:
I'd go further with the edit pen and say something like

"At the time of writing, two application bindings for QUIC have been defined by the IETF: HTTP/3 {{?QUIC-HTTP}} and {{?I-D.draft-ietf-dprive-dnsoquic}}. These are both known to have active internet deployments, so an assumption that all"

Reply:
sgtm, will do.